src/megaapi_impl.cpp

```diff
@@ -3942,6 +3973,17 @@ char *MegaApiImpl::getMyXMPPJid()
     return result;
 }
 
+bool MegaApiImpl::isAchievementsEnabled()
+{
+    bool ret;
+
+    sdkMutex.lock();
+    ret = client->achievements_enabled;
+    sdkMutex.unlock();
+
```

Comment:
I think that we don't need the mutex here.

Reply:
Removed the mutex lock/unlock